src/style.css

```diff
@@ -0,0 +1,52 @@
+:root {
+  --not-active: url("images/star.svg");
+  --active: url("images/star-active.svg");
+}
+
+* {
+  margin: 0;
```

Comment:
you shouldn't reset indentation for the * selector, do it directly to selector you need to reset